README.md

```diff
@@ -29,7 +29,40 @@ It still might work on older rust versions, but this isn't ensured.
 
 The following diagram illustrates the user-facing and internal crates of the defmt framework.
 
-![defmt crates structure](assets/defmt.png)
+```mermaid
+%%{ init: { 'flowchart': { 'curve': 'step', "defaultRenderer": "elk" } } }%%
```

Comment:
`"defaultRenderer": "elk"` seems to be the culprit for the `~~~` invisible link not working.